amundsen-kube-helm/templates/helm/values.yaml

```diff
@@ -307,8 +307,8 @@ neo4j:
       - "/bin/sh"
       - "-c"
       - |
-        curl -L https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases/download/3.3.0.4/apoc-3.3.0.4-all.jar -O
-        curl -L https://github.com/neo4j-contrib/neo4j-graph-algorithms/releases/download/3.3.5.0/graph-algorithms-algo-3.3.5.0.jar -O
+        curl -L https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases/download/3.5.0.17/apoc-3.5.0.17-all.jar -O
```

Comment:
Do we need to set up the plugins? I see that the official images already have them [installed](https://github.com/neo4j/docker-neo4j-publish/blob/master/3.5.26/community/local-package/neo4jlabs-plugins.json)?

Reply:
Hi @MrwanBaghdad , thanks a lot for the quick review!

According to this [discussion](https://github.com/amundsen-io/amundsen/discussions/1801#discussioncomment-2512639), these plugins don't seem to be installed by default. Also, since the plugins install curl are already there originally, I think the safest way is to just update it to the compatible version. I could verify that this works just fine 💯